x-pack/solutions/security/plugins/security_solution/docs/testing/test_plans/detection_response/prebuilt_rules/prebuilt_rule_import.md

```diff
@@ -78,70 +79,85 @@ https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one
 
 ```Gherkin
-Given the import payload contains a prebuilt rule with a matching rule_id and version, identical to the published rule
+Given the import payload contains a non-customized prebuilt rule
+And its rule_id and version match a rule asset from the installed package
 When the user imports the rule
 Then the rule should be created or updated
 And the ruleSource type should be "external"
 And isCustomized should be false
 ```
 
-#### Scenario: Importing a customized prebuilt rule with a matching rule_id and version
+#### Scenario: Importing a customized prebuilt rule with a matching `rule_id` and `version`
 
 **Automation**: 1 cypress test and 1 integration test.
 
 ```Gherkin
-Given the import payload contains a prebuilt rule with a matching rule_id and version, modified from the published version
-And the overwrite flag is set to true
+Given the import payload contains a customized prebuilt rule
+And its rule_id and version match a rule asset from the installed package
 When the user imports the rule
 Then the rule should be created or updated
 And the ruleSource type should be "external"
 And isCustomized should be true
+```
+
+#### Scenario: Importing a custom rule with a matching prebuilt `rule_id` and `version`
+
+**Automation**: 1 cypress test and 1 integration test.
 
-CASE: Should work with older, newer, or identical version numbers
+```Gherkin
+Given the import payload contains a custom rule
+And its rule_id and version match a rule asset from the installed package
+When the user imports the rule
+Then the rule should be created or updated
+And the ruleSource type should be "external"
+And isCustomized should be true
 ```
 
-#### Scenario: Importing a custom rule with a matching rule_id and version
+#### Scenario: Importing a custom rule with a matching custom `rule_id` and `version`
 
 **Automation**: 1 cypress test and 1 integration test.
 
 ```Gherkin
-Given the import payload contains a custom rule with a matching rule_id and version
+Given the import payload contains a custom rule
+And its rule_id and version match a rule asset from the installed package
 And the overwrite flag is set to true
 When the user imports the rule
-Then the rule should be updated
+Then the rule should be created or updated
 And the ruleSource type should be "internal"
 ```
 
-#### Scenario: Importing a prebuilt rule with a matching rule_id but no matching version
+#### Scenario: Importing a prebuilt rule with a matching `rule_id` but no matching `version`
 
 **Automation**: 1 integration test.
 
 ```Gherkin
-Given the import payload contains a prebuilt rule with a matching rule_id but no matching version
-And the overwrite flag is set to true
+Given the import payload contains a prebuilt rule
+And its rule_id matches one or a few rule assets from the installed package
+And its version does NOT match any of those rule assets
 When the user imports the rule
-Then the rule should be created
+Then the rule should be created or updated
```

Comment:
We should have separate, more specific scenarios for each scenario that says `should be created or updated`. This assumes different test setups and can't be covered by a single integration test.